drivers/include/ds18.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2017 Frits Kuipers
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_ds18 DS18 temperature sensor driver
+ * @ingroup     drivers_sensors
+ * @ingroup     drivers_saul
+ * @brief       Driver interface for the DS18 temperature sensors
+ *
+ * This driver provides @ref drivers_saul capabilities.
```

Comment:
Would you please also add some more documentation about the supported (or not supported) features and limitations of this driver?!

E.g.
- does not allow for addressing devices -> only supports a single device on the bus
- hardcodes the 1wire bus functionality into the driver
- does not allow for configuration of the sampling width